README.md

```diff
@@ -14,20 +14,20 @@
 <a href="https://crates.io/crates/av1an"><img src="https://img.shields.io/crates/v/av1an.svg"></a>
 
 </h4>
-<h2 align="center">Easy, Fast, Efficient and Feature Rich</h2>
+<h2 align="center">Easy, Fast, Efficient and Feature-Rich</h2>
 
-### <center>An easy way to start using AV1 / HEVC / H264 / VP9 / VP8 encoding. AOM, RAV1E, SVT-AV1, VPX, x265, x264 are supported</center>
+### <center>An easy way to start using AV1, HEVC/H.265, AVC/H.264, VP9, and VP8 encoders.<br>Encoders such as AOM, RAV1E, SVT-AV1, VPX, x265, x264 are supported.</center>
```

Comment:
Personally I find the "such as" wording to be a little confusing, since "such as" usually implies that we are listing a subset of something, when in fact here we are listing all supported encoders.

Reply:
So what should be the better replacement for it?

Reply:
I think just removing the "Encoders such as" part would be fine. Also, I noticed you added a `<br>` tag that seems to be missing the corresponding closing tag (`</br>`)? I don't think the tag is necessary in the first place either since to my knowledge, making a heading (with `###`) already makes the text bold.

Reply:
Ah nevermind, for some reason I got confused and mixed up `<br>` and `<b>`.